src/sdk/pynni/nni/nas/pytorch/random/mutator.py

```diff
@@ -0,0 +1,25 @@
+import torch
```

Comment:
license?

Reply:
License will be included in a PR after cutting branch, as discussed before.